frontend/packages/network-attachment-definition-plugin/src/hooks/form-validation-hook.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { networkTypeParams } from '../constants';
+
+export const useFormValidationHook = (fieldErrors, name, networkType, typeParamsData) => {
```

Comment:
this does not have to be custom hook, it can be just a function which will be used in `React.useCallback`

Reply:
As usual, you're right. =) Done!